truffle-config.js

```diff
@@ -1,5 +1,16 @@
 require('chai/register-should');
 
+const solcStable = {
+  version: '0.5.7',
+};
+
+const solcNightly = {
+  version: 'nightly',
+  docker: true,
```

Comment:
Is docker a requirement for the nightly build?

Reply:
Yes, it's the only way I found to use a nightly build, they aren't available using solcjs with Truffle.

Reply:
Neat, this shouldn't cause any issues for users anyway.